src/nccl_ofi_rdma.cpp

```diff
@@ -903,20 +894,16 @@ static inline int update_send_data_from_remote(nccl_net_ofi_rdma_send_comm_t *s_
 	nccl_net_ofi_scheduler_t *scheduler = domain->scheduler;
 
 	rdma_req_send_data_t *send_data = get_send_data(req);
-	rdma_req_rx_buff_data_t *rx_buff_data = get_rx_buff_data(rx_buff_req);
-	nccl_net_ofi_rdma_ctrl_msg_t *ctrl_msg = get_rx_ctrl_msg(rx_buff_data);
+
+	uint16_t slot = s_comm->next_msg_seq_num % NCCL_OFI_MAX_REQUESTS;
```

Comment:
I don't love the abstraction here.  It seems better to pass in the slot explicitly in place of the rx_buff_req (which we shouldn't need).